src/main/resources/lib/enonic/react4xp/dependencies.es6

```diff
@@ -155,12 +154,15 @@ const readComponentChunkNames = entryNames => {
     return output;
 }
 
+
+
 // Cached version of readComponentChunkNames - used in prod mode
-const readComponentChunkNamesCached = entryNames => {
-  entryNames = normalizeEntryNames(entryNames);
-  const entryNamesKey = entryNames.join("*");
+const readComponentChunkNamesCached = (entryNames, siteId) => {
+    siteId = siteId || getSite()._id;
```

Comment:
if (!siteId) {
siteId = getSite()._id;
}

Reply:
In fact this is already done inside getComponentChunkUrls. And readComponentChunkNamesCached is not exported so it's useless?